networkit/_NetworKit.pyx

```diff
@@ -6158,11 +6171,25 @@ cdef class ConnectedComponents(Algorithm):
 		component_sizes = cc_result.component_sizes
 		n_components = cc_result.n_components
 		array_wrapper = ArrayWrapper()
-		np_mapping_array = array_wrapper.as_ndarray(n_nodes, <void *>components)
+		np_mapping_array = array_wrapper.as_ndarray(n_nodes, <void *>components, np.NPY_UINT64)
```

Comment:
I would rather use `ArrayWrapper()` inline instead of redefining `np_mapping_array` many times:
```suggestion
		np_mapping_array = ArrayWrapper().as_ndarray(n_nodes, <void *>components, np.NPY_UINT64)
```